packages/eslint-config-base/rules/imports.js

```diff
@@ -93,6 +93,14 @@ module.exports = {
         ],
       },
     ],
+    /**
+     * Use named exports over default exports
+     * @link https://github.com/reside-eng/guidelines/blob/main/rfcs/shared-patterns/typescript/001-consistent-import-export.md
+     */
+    ...{
+      'import/prefer-default-export': 'off',
+      'import/no-default-export': 'error',
+    },
```

Comment:
```suggestion
    /**
     * Use named exports over default exports
     * @link https://github.com/reside-eng/guidelines/blob/main/rfcs/shared-patterns/typescript/001-consistent-import-export.md
     */
    'import/prefer-default-export': 'off',
    'import/no-default-export': 'error',
```

Reply:
Let's just group these two rules next to one another without an extra line break. We don't need to spread an object literal just so we can encapsulate the comment.